tests/cases/conformance/jsx/inline/inlineJsxFactoryWithFragmentIsError.tsx

```diff
@@ -0,0 +1,14 @@
+// @jsx: react
+// @filename: renderer.d.ts
+declare global {
+    namespace JSX {
+        interface IntrinsicElements {
+            [e: string]: any;
+        }
+    }
+}
+export function dom(): void;
+// @filename: index.tsx

```

Comment:
also add a test with two files, and `@jsxNamespace` defined, and one of them with the new pragma.